### PR TITLE
For mozilla-mobile/fenix#5540: add 'no internet' error

### PR DESCRIPTION
**Pull request by Baron-Severin**
---

@Amejia481 🤦‍♂  closed the other issue too quickly.  I do still need AC changes.  See https://github.com/mozilla-mobile/android-components/pull/5068#issuecomment-558415367

### Pull Request checklist

- [X] **Quality**: This PR builds and passes detekt/ktlint checks (A pre-push hook is recommended)
- [X] **Tests**: This PR includes thorough tests or an explanation of why it does not
This only makes changes to copy.
- [X] **Changelog**: This PR includes [a changelog entry](https://github.com/mozilla-mobile/android-components/blob/master/docs/changelog.md) or does not need one
- [X] **Accessibility**: The code in this PR follows [accessibility best practices](https://github.com/mozilla-mobile/shared-docs/blob/master/android/accessibility_guide.md) or does not include any user facing features

### After merge
- [ ] **Milestone**: Make sure issues closed by this pull request are added to the [milestone](https://github.com/mozilla-mobile/android-components/milestones) of the version currently in development.
- [ ] **Breaking Changes**: If this is a breaking change, please push a draft PR on [Reference Browser](https://github.com/mozilla-mobile/reference-browser) to address the breaking issues.

Screen Shot 2019-11-14 at 4 59 05 PM

### CODE REVIEW

File: components/browser/errorpages/src/main/java/mozilla/components/browser/errorpages/ErrorPages.kt [hunk 1]
```diff
@@ -45,7 +45,8 @@ object ErrorPages {
  */
 enum class ErrorType(
     @StringRes val titleRes: Int,
-    @StringRes val messageRes: Int
+    @StringRes val messageRes: Int,
+    @StringRes val refreshButtonRes: Int = R.string.mozac_browser_errorpages_page_refresh
```
Comment (Amejia481) [inline components/browser/errorpages/src/main/java/mozilla/components/browser/errorpages/ErrorPages.kt]: nit: I see that we are adding this (`refreshButtonRes`) property but we are not using it, is there any reason?

Comment (Baron-Severin) [reply components/browser/errorpages/src/main/java/mozilla/components/browser/errorpages/ErrorPages.kt]: Whoops!  Fixed.

File: docs/changelog.md [hunk 2]
```diff
@@ -54,6 +54,9 @@ permalink: /changelog/
     * String list items were being truncated to 20, rather than 50, bytes when using
       `.set()` (rather than `.add()`). This has been corrected, but it may result
       in changes in the sent data if using string list items longer than 20 bytes.
+      
+* **browser-errorpages**
```
Comment (Amejia481) [inline docs/changelog.md]: nit: Let's move this entry to the # 24.0.0-SNAPSHOT section :)

**Comment by Amejia481** [comment]
bors retry

**Pull request merged**